### ISSUE TITLE
Some error in using CASSCF

### ISSUE THREAD

**Issue by newuserofpyscf** [pyscf/pyscf#1739]
HI, every one , I am new to quamtum chemistry and PYSCF. I just tried to use PYSCF to make a  casscf pactice. But some error occur 
![1](https://github.com/pyscf/pyscf/assets/133973641/8eea9138-52c1-40d4-9aa3-9063b103f23a)
![2](https://github.com/pyscf/pyscf/assets/133973641/2ee5b66e-5306-4a0e-a8f5-be05fdd791eb)
![3](https://github.com/pyscf/pyscf/assets/133973641/7a43e269-7100-4877-8de0-b2a859cdf92f)

Could anyone help me to solve this problem, please

**Comment by hebrewsnabla** [comment]
`myhf = scf.ROHF(mol).kernel()` should be replaced by
```
myhf = scf.ROHF(mol).run()
```
or 
```
myhf = scf.ROHF(mol)
myhf.kernel()
```

**Comment by newuserofpyscf** [comment]
> `myhf = scf.ROHF(mol).kernel()` should be replaced by
> 
> ```
> myhf = scf.ROHF(mol).run()
> ```
> 
> or
> 
> ```
> myhf = scf.ROHF(mol)
> myhf.kernel()
> ```

Thank you so much for your reply! 

Ummm, Is this a syntax about Python？How much knowledge of Python do I need to acquire, in order to learn PySCF?

**Comment by hebrewsnabla** [comment]
No. It's a convention of PySCF. The `kernel()` function and `run()` do the same calculation, but `run()` returns a SCF object while `kernel()` not. So you cannot let `myhf = scf.ROHF(mol).kernel()`.

Yeah, you need a little python knowledge to understand the documentation.

**Comment by verena-neufeld** [comment]
Hi @newuserofpyscf , thanks for using pyscf and for posting this question. Has it been answered?
